checkout/react/component/Menu/Desktop/styles.scss

```diff
@@ -152,61 +259,105 @@
     }
   }
 }
+
 .mega-menu__brands-title {
-    margin: 12px 0;
-    padding-bottom: 8px;
-    color: #757d83 !important;
-    font-weight: 700 !important;
-    border-bottom: 1px solid #dbdbdb;
+  margin: 12px 0;
+  padding-bottom: 8px;
+  color: #757d83 !important;
+  font-weight: 700 !important;
+  border-bottom: 1px solid #dbdbdb;
 }
 
 .categories_group__subcate.gray {
   background-color: #f2f0f0;
+  padding-bottom: 32px;
 }
+
 .categories_group__subcate.white {
   background-color: #fff;
+  display: flex;
 }
 
 .container-brands {
-    width: 100%;
+  width: 100%;
+  margin: 16px 10% 0 0;
 }
-.container-cate{
-    width: 60%;
-    display: flex;
-    margin: 20px 10% 0;
-    // border-right: 1px solid #dbdbdb;
+
+.container-cate {
+  width: 100%;
+  display: flex;
+  max-width: 594px;
 }
 
 .categories_group__subcate {
-  display: block;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: inline-block;
+  -webkit-box-align: flex-start;
+  -ms-flex-align: flex-start;
+  align-items: flex-start;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  margin-left: 0;
   width: 100%;
-  margin: 0;
-  display: block;
+  max-width: 236px;
+  margin-top: 16px;
+  position: relative;
 
   .categories_group__subcate_img {
-    justify-content: center;
-    align-items: center;
-    grid-gap: 10px;
-    width: 33%;
-    display: inline-block;
-    vertical-align: top;
-    padding: 12px 16px;
+    margin: 12px 0;
+    width: calc((100% - 40px) / 3);
+    -webkit-filter: grayscale(1);
     filter: grayscale(1);
+
     img {
       margin: auto;
     }
   }
 
-  li {
+  .categories_group__subcate_all-brands {
+    font-weight: 700;
+    text-transform: uppercase;
+    font-size: 1.3rem;
+    padding: 12px 16px;
+    list-style: none;
+    display: flex;
+
+    &::after {
+      content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 477.175 477.175' class='svg-icon'%3E%3Cpath d='M360.731 229.075l-225.1-225.1c-5.3-5.3-13.8-5.3-19.1 0s-5.3 13.8 0 19.1l215.5 215.5-215.5 215.5c-5.3 5.3-5.3 13.8 0 19.1 2.6 2.6 6.1 4 9.5 4 3.4 0 6.9-1.3 9.5-4l225.1-225.1c5.3-5.2 5.3-13.8.1-19z'%3E%3C/path%3E%3C/svg%3E");
```

Comment:
Same here with regards to putting it in assets?

Reply:
Done, an assets/icons folder was created